intro/checkAge.js

```diff
@@ -1,7 +1,4 @@
 function checkCondition(people) {
```

Comment:
function name doesn't tell what it does/check! Suggested, name for the function: `checkAge`, `checkAgeInObject`, etc.